css/main.css

```diff
@@ -24,6 +24,7 @@
 	--colour-warning-third: #624711;
 	--colour-error: #f8513d;
 	--colour-error-second: #6b2a2b;
+	--colour-black-with-opacity: rgba(0, 0, 0, 0.5);
```

Comment:
instead of black dark word could be used and with could be omited

Reply:
sure will make those changes